### ISSUE TITLE
Remove pep8 for PR.

### ISSUE THREAD

**Issue by samuelgarcia** [NeuralEnsemble/python-neo#567]
We activated pep8speak this January on every PR.

My personnal feedback on this is that it is counter productive for reviewing new PR.
Many PR involve few lines and reviewing them should be fast but now in some PR 80%  patched lines are linked to pep8 and not the logic itself.

Commits are sometimes very hard to follow when several files are involved.

I am happy that neo is pep8 compatible but forcing it for PR lead to long (and bad for me) reviews.

Maybe we could stop pep8speak and before each release we could clean with an autopep8.

I propose a long, democratic and painfull debate on this and then a dictatorial decission in a small comitee... (too bad).

**Comment by JuliaSprenger** [comment]
To starting the democratic discussion:
For the majority of PRs, where just a bit of code changed and after feedback via pep8speaks the layout is adjusted I don't think it harms too much, because the different changes are split into different commits, which can be viewed separately.
For large PRs involving multiple files and lots of lines of code it is anyway necessary to review a lot of changes, so it does not make a big difference.
The disadvantage of having a big autopep8 PR is that you still want to make sure all the changes look (somewhat) ok, meaning you have to read the complete Neo code.
Also in the long run the remarks by pep8speaks should reduce in number, since a slowly all files should get more compatible to pep8. I am actually surprised this did not happen after the last Neo release, which included also an autopep8 PR.

**Comment by achilleas-k** [comment]
If I can chime in with a minor comment: If it's possible to make pep8speaks not comment when there are no issues, that might be a nice small step towards reducing unnecessary chatter.

Another solution would be a checker like CircleCI or Travis that puts a checkmark and links to results instead of putting them in the comment thread. I looked around and couldn't find a dedicated one, but maybe a Travis build could be added that simply runs a pep8 check script and succeeds or fails accordingly.

**Comment by samuelgarcia** [comment]
Could we at least remove the E127 check "continuation line over-indented for visual indent" ?

**Comment by apdavison** [comment]
After discussion in Nov 15th code sprint, agreed to keep pep8speaks, but configure it to relax some of the restrictions.
